M2/Macaulay2/m2/classes.m2

```diff
@@ -22,7 +22,7 @@ pluralize = s -> demark_" " append(
     drop(ws := separate_" " s, -1),
     if  plurals#?(last ws)
     then plurals#(last ws) else last ws | "s")
-pluralsynonym = T -> if T.?synonym then pluralize T.synonym else "objects of class " | toString T
+pluralsynonym = T -> pluralize lookup(symbol synonym,T) -- pluralize T.synonym ?? "objects of class " | toString T
```

Comment:
1. Were the changes to `synonym` and `pluralsynonym` included by accident?
2. What happens if a type doesn't have a synonym? I prefer `object of class X` to just null.
3. Assuming this change is desired, `pluralsynonym = T -> pluralize synonym T` seems simpler.

Reply:
just to clarify, this PR is very much work in progress. This will be reverted, it was just an attempt.

Reply:
Ah, sorry I missed that this was a draft.